linalg/mma.cpp

```diff
@@ -11,9 +11,23 @@
 
 #include "mma.hpp"
 
+#include "vector.hpp"
+#include "../general/communication.hpp"
+#include "../general/error.hpp"
+
 #include <fstream>
 #include <math.h>
 
+namespace
+{
+// check if C++ 14 or beyond
```

Comment:
One thing to note: the current MFEM standard is C++11.  (We plan to switch to C++17 later this year.)

Reply:
> One thing to note: the current MFEM standard is C++11. (We plan to switch to C++17 later this year.)

The `#if` should make it work with either C++-11, C++-14, C++-17, or beyond.